steelthread-0/walkthrough.md

```diff
@@ -15,123 +18,234 @@ The installation has already been performed on an OpenShift cluster as per the i
-- Choose the integration name (steel thread)
+- Click on the ```Subscribe for messages``` option
+- Under Destination Name add ```work-queue/requests ```
+- Choose `Queue` as the Destination Type
+- Click `Next`
+- Set the `Select Type` as ```JSON Schema``` 
+- Add the following schema to the `Definition` field & click `Done`
+```
+{
+	"$schema": "http://json-schema.org/draft-04/schema#",
+	"type": "object",
+	"properties": {
+		"type": {
+			"type": "string"
+		},
+		"stock": {
+			"type": "string"
+		}
+	}
+}
+```
+- On the ```Choose a Finish Connection``` screen select the `crud booster` connection you created
+- Choose ```Invoke URL``` 
+- In the URL Path set the path to ```/api/fruits```
+- In the method select ```POST``` and click next
+- Again under `Select Type` set it to ```JSON Schema``` 
+- Add the following schema to the `Definition` field & click `Done`
+```
+{
+	"$schema": "http://json-schema.org/draft-04/schema#",
+	"type": "object",
+	"properties": {
+		"name": {
+			"type": "string"
+		},
+		"stock": {
+			"type": "string"
+		}
+	}
+}
+```
+
+- Next click the small plus between the two connections on the left hand side then click ```Add a step```
+- Click ```Data Mapper``` 
+- Map the `stock` property in the Source to the `stock` property in the Target by dragging from one side to the other. If done correct, a blue line will link the Source & Target.
+- Map `type` in the Source to `name` in the Target, again by dragging from one side to the other
+- Click `Done` at the top right
+
+## Add some Integration logging steps (optional)
+
+To add some more visibility we can add some logging steps to our integration.
+
+- On the left hand side inbetween the connections you will see small blue boxes with plus symbols. For each of these do the following:
+
+- click the blue plus symbol
+- click `Add a step`
+- choose `Log`
+- select ```Message Body```
+
+Once done we should now have 3 steps (5 if you added logging) in our integration. Click `Publish` in the top right.
+
+Name the integration ```steel thread```
+
+
+# Setup CHE to enable editing of the booster
+
+The Che Dashboard can be found at ```https://che-che.<ROUTER_SUFFIX>``` 
+
+## Self Signed Certs setup
+
+If using a self signed (non CA) certificate on the Che route, there are a few things that need to be done before a Workspace can be created and used.
+
+* Import the cert into Che Server (this is done autmatically by the ansible installer)
+* Create a custom Node.js stack with the self signed certificate included. This is required so that stack instances (workspaces) can make requests back to the Che server.
+* Import the self signed certificate into your browser. This is required so that a websocket connection back to the Che Server can be established, which is required by many parts of the Che Browser IDE.
+
+Docs for how to do items 2 & 3 are available at https://www.eclipse.org/che/docs/openshift-config.html.
+
+## Authorize Che against your Github Account
```

Comment:
I don't think is required any longer as Che uses the Launcher keycloak so Github oauth has already been authorized